### DISCUSSION TITLE
Update env

### DISCUSSION THREAD

**Discussion by jhihyulin** [deta/cloud-docs#497] [Help]
`deta update -e .env`
When I used it for the first time, it was normal as shown below
![Screenshot_2022-09-22-16-42-49-81_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/74854953/191701560-1f09e90d-1d9b-47a0-bdc0-2d9216ce1d3b.jpg)
But on the second time to update environment variable default value disappears
![Screenshot_2022-09-22-16-43-55-98_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/74854953/191701748-801a6fa7-f23b-4ba5-808b-6a043ea330af.jpg)

**Comment by lemonyte** [comment]
This is likely a bug in the UI. The default environment variables are still there.
